src/Container/Errors.hs

```diff
@@ -35,3 +39,16 @@ instance ToDiagnostic ContainerImgParsingError where
 
 instance ToDiagnostic (NonEmpty ContainerImgParsingError) where
   renderDiagnostic = pretty . show
+
+data EndpointDoesNotSupportNativeContainerScan = EndpointDoesNotSupportNativeContainerScan
+instance ToDiagnostic EndpointDoesNotSupportNativeContainerScan where
+  renderDiagnostic (EndpointDoesNotSupportNativeContainerScan) =
+    vsep
+      [ "Provided endpoint does not support native container scans."
+      , ""
+      , "If you are using, --experimental-scanner option, it is not supported for your"
+      , "FOSSA instance. Try without using --experimental-scanner."
```

Comment:
We want to avoid a scenario where the on-prem customer uses the latest cli and experimental function. Still, their FOSSA instance is not configured or does not have the necessary functionality. 

Historically, we have been using `/api/cli/org..` endpoint to communicate this information. In this endpoint, we also return the status of the feature flag or general capability information (fixed via boolean value).

Reply:
The reason I ask is because if I were a SaaS user and saw this then it would read like FOSSA SaaS doesn't support the experimental scanner which would be strange. Should it say "FOSSA instance or organization"?

Would a SaaS customer ever see this error?

Reply:
Yeah, SaaS customers should never see this error. I think Fossa instance is more accurate, as it is applied to all organization (just how it is implemented in core: https://github.com/fossas/FOSSA/pull/8174)

Reply:
That makes sense to me, I misunderstood who this error was for. Thanks for clarifying!